src/shared/Atlassian.Bitbucket.UI.Avalonia/Program.cs

```diff
@@ -45,9 +45,15 @@ private static void AppMain(object o)
         {
```

Comment:
I never use Avalonia, so I'm just going to ask a few questions.

Does the Main() and/or AppMain() function run in a separate process from the GCM proper process?

Does this new avalonia process need/want a new SID that is a child of the GCM SID?  I can't tell here
whether the New CommandContext() or context.Tract2.Initialize() generates or calls the SID Manager
class (like we do in the updated GCM Main()).

Reply:
I'm fine with giving them child Sids. It will mean removing the guard we spoke about previously, but that's not a big deal.